src/components/app/data/queries/queryKeyFactory.js

```diff
@@ -151,5 +152,17 @@ const user = createQueryKeys('user', {
   }),
 });
 
-const queries = mergeQueryKeys(enterprise, user);
+const program = createQueryKeys('program', {
+  progress: {
+    queryKey: null,
+    contextQueries: {
+      detail: (programUUID) => ({
```

Comment:
[question/suggestion] Given the `programUUID` will also be relevant for the program detail page route, is there a way to set up the query keys such that `programUUID` can be specified in one place in the query key factory, but apply to both the program progress AND program detail? As is, I believe you'd have 2 queries both accepting a `programUUID` which could be simplified a bit? 🤔 

Perhaps treating these query keys as `content` vs. `program`, such that the initial key can accept a function which accepts `programUUID` as an argument:

```jsx
const content = createQueryKeys('content', {
  program: (programUUID) => ({
    queryKey: [programUUID],
    contextQueries: {
      detail: {
        queryKey: null,
        queryFn: () => {}, // placeholder for WIP program detail PR
      },
      progress: {
        queryKey: null,
        queryFn: async ({ queryKey }) => fetchLearnerProgramProgressDetail(queryKey[3]), // `queryKey` index of 3 likely would change?
      },
    },
  }),
});
```